.editorconfig

```diff
@@ -3,6 +3,9 @@
 # top-most EditorConfig file
 root = true
 
+[*]
```

Comment:
To me, it makes sense to keep this at the "all" level; I don't think anything in our repository intentionally uses trailing whitespace.